app/uk/gov/hmrc/helptosavereminder/repo/HtsReminderRepository.scala

```diff
@@ -169,27 +159,29 @@ class HtsReminderMongoRepository @Inject()(mongo: ReactiveMongoComponent)
   override def updateReminderUser(htsReminder: HtsUser): Future[Boolean] = {
 
     val selector = Json.obj("nino" -> htsReminder.nino.nino)
+    val modifierJson = Json.obj(
+      "optInStatus"   -> JsBoolean(htsReminder.optInStatus),
+      "email"         -> htsReminder.email,
+      "firstName"     -> htsReminder.firstName,
+      "lastName"      -> htsReminder.lastName,
+      "nextSendDate"  -> getNextSendDate(htsReminder.daysToReceive),
+      "daysToReceive" -> htsReminder.daysToReceive
+    )
+
+    val updatedModifierJson =
```

Comment:
Minor niggle, I always notice "if(!" and to my mind it isn't helped by the negative nature of isEmpty.

Personally, I would at least remove the ! and reorder the terms around the else, but it is up to you.

I do wonder if there is a batter way to construct the map that goes into Json.obj without an if, but one doesn't come to mind straight away

Reply:
Fine, no probs.

Reply:
Done